azure-pipelines-tests-template.yml

```diff
@@ -0,0 +1,18 @@
+parameters:
```

Comment:
can we name this file `azure-pipelines-e2e-tests-template.yml` instead ?

Reply:
@suhsteve good idea. I've renamed the file.